### PR TITLE
Maven dependencies updated

### PR DESCRIPTION
**Pull request by MohammedAman23**
maven dependencies in pom updated

### CODE REVIEW

**Comment by karun-singh** [comment]
Error log from the failed Unit test stage in the CICD pipeline - `[ERROR] /usr/share/app/src/test/java/iudx/aaa/server/policy/CreateUserPolicyTest.java:[26,23] package junit.framework does not exist`
- Update the import there as well

**Comment by karun-singh** [comment]
Error in failing unit tests- `java.lang.NoSuchMethodError: 'void com.google.common.base.Preconditions.checkArgument(boolean, java.lang.String, java.lang.Object)'`
https://jenkins.iudx.io/job/iudx%20aaa-server%20(PR)%20pipeline/221/testReport/

**Comment by ThorodanBrom** [comment]
retest this please

**Comment by ThorodanBrom** [comment]
I read that this error can be caused by [multiple versions of Guava being present in the project/classpath](https://issues.apache.org/jira/browse/HIVE-22915). This may be applicable here:

Screenshot_2022-09-29_17-58-43

We can try downgrading guava back to the old version and see

**Comment by MohammedAman23** [comment]
retest this please

**Comment by MohammedAman23** [comment]
retest this please

**Comment by ThorodanBrom** [comment]
retest this please

File: Jenkinsfile [hunk 1]
```diff
@@ -84,16 +84,18 @@ pipeline {
...
-            publishHTML([allowMissing: false, alwaysLinkToLastBuild: true, keepAll: true, reportDir: '/var/lib/jenkins/iudx/aaa/Newman/report/', reportFiles: 'report.html', reportName: 'HTML Report', reportTitles: '', reportName: 'Integration Test Report'])
+
           }
         }
         failure{
           error "Test failure. Stopping pipeline execution!"
         }
         cleanup{
           script{
+            docker logs iudxaaa-serverprpipeline_integTest_1 > /tmp/aaa.logs
```
Comment (karun-singh) [inline Jenkinsfile:98]:
```suggestion
            sh 'docker logs iudxaaa-serverprpipeline_integTest_1 > /tmp/aaa.logs'
```

**Comment by ThorodanBrom** [comment]
Closing, made new branch #253 for this

**Pull request closed without merging**